components/brave_sync/brave_profile_sync_service.cc

```diff
@@ -484,8 +484,12 @@ void BraveProfileSyncService::OnSyncReady() {
   if (sync_prefs.GetLastSyncedTime().is_null()) {
     ProfileSyncService::GetUserSettings()
       ->SetSelectedTypes(false, syncer::UserSelectableTypeSet());
+    syncer::UserSelectableTypeSet type_set =
+        ProfileSyncService::GetUserSettings()->GetSelectedTypes();
+    type_set.Remove(syncer::UserSelectableType::kBookmarks);
+    ProfileSyncService::GetUserSettings()->SetSelectedTypes(false, type_set);
```

Comment:
You can just use an empty set here because before sync chain established, all the data except device records might be missing forever.

Reply:
Thanks, had removed this.